source/utils.ts

```diff
@@ -3,20 +3,50 @@ export type JSONObject = { [key: string]: JSON };
 
 type UnknownRecord = Record<string, unknown>;
 
-export function get(object: UnknownRecord, path: string) {
+const getTranslationWithCount = (
+  originalCount: number,
+  index: number,
+  translation: string,
+) => {
+  const options = translation.split(' | ');
+  const pluralized =
+    options[index >= options.length ? options.length - 1 : index];
+
+  if (pluralized.includes('{count}')) {
+    return pluralized.replace('{count}', `${originalCount}`);
```

Comment:
Maybe use `replaceAll`. It will more versatile.

Reply:
What gains do you see in using `replaceAll`? Current `replace` will replace only one instance of `{$count}`, and there should be only one. So using `replaceAll` may obscure potential bugs.